operational_analysis/methods/plant_analysis.py

```diff
@@ -655,28 +669,39 @@ def run_AEP_monte_carlo(self):
 
         num_sim = self.num_sim
 
-        sim_results = pd.DataFrame(index=np.arange(num_sim), data={'aep_GWh': np.empty(num_sim),
-                                                                   'avail_pct': np.empty(num_sim),
-                                                                   'curt_pct': np.empty(num_sim)})
+        aep_GWh = np.empty(num_sim)
```

Comment:
Improve simulation speed by ~25% by using numpy arrays instead of pandas dataframe.